remappings.txt

```diff
@@ -1,4 +1,3 @@
 forge-std/=lib/forge-std/src/
 @chainlink/=lib/chainlink-brownie-contracts/contracts/src/
```

Comment:
The`/out` folder was pushed intentionally, I did this to minimize the actions required by the user for the plugin to work (downloading toolkit's submodules and building artifacts)

Reply:
> It would be cool to set remappings in a way that the user will import Chainlink contracts using the following prefix: `@chainlink/contracts/...`

Remappings were updated according to the suggestion.